openmdao/core/problem.py

```diff
@@ -642,12 +638,12 @@ def check_partials(self, logger=None, comps=None, compact_print=False,
 
                             # Testing for pairs that are not dependent so that we suppress printing
                             # them unless the fd is non zero. Note: subjacs_info is empty for
-                            # undeclared partials on implicit components.
+                            # undeclared partials, which is the default behvaior now.
```

Comment:
typo "behvaior"... also, the "now" is not really necessary

Reply:
will fix on justin's PR